app/_hub/kong-inc/kafka-log/index.md

```diff
@@ -238,5 +167,19 @@ The following guidelines assume that both `Kong` and `Kafka` have been installed
         --timeout-ms 1000
     ```
 
+## Log format
+
+Similar to the [HTTP Log Plugin](https://docs.konghq.com/hub/kong-inc/http-log#log-format).
+
+## Implementation details
+
+This plugin makes use of the [lua-resty-kafka](https://github.com/doujiang24/lua-resty-kafka) client under the hood.
+
+## Known issues and limitations
+
+1. There is no support for TLS.
+2. There is no support for authentication.
+3. There is no support for message compression.
+
 [badge-travis-url]: https://travis-ci.com/Kong/kong-plugin-kafka-log/branches
```

Comment:
not sure we need these links. these comments will be addressed in a subsequent ticket.